### PR TITLE
fix: correct fields and add schema

### PR DESCRIPTION
**Pull request by RamyEB**

### CODE REVIEW

File: packages/manifest-validator/src/schema/schema.ts [hunk 1]
```diff
@@ -68,8 +72,16 @@ export const schema = {
...
-      format: "uri",
+      oneOf: [
+        {
+          type: "string",
+          format: "uri",
+        },
+        {
+          type: "string",
+          maxLength: 0,
+        },
```
Comment (Justkant) [inline packages/manifest-validator/src/schema/schema.ts:83]: Is it to make the field icon optional or support `""` ?
We probably should remove the field instead if a manifest have an icon as an empty string, wdyt ?

**Comment by Wozacosta** [comment]
> [remove uri format causing errors](https://github.com/LedgerHQ/wallet-api/pull/353/commits/e4d97018694ad203208a28f44e5c7612f4baba29) 


What kind of errors did it give that it warrant a complete removal of the type? Was it too cumbersome?

**Comment by Justkant** [comment]
You'll need a changeset too for this to be released

**Comment by RamyEB** [comment]
> > [remove uri format causing errors](https://github.com/LedgerHQ/wallet-api/pull/353/commits/e4d97018694ad203208a28f44e5c7612f4baba29)
> 
> What kind of errors did it give that it warrant a complete removal of the type? Was it too cumbersome?

Added back cause i used ajv without extra format :
unknown format "uri" ignored in schema at path "#/properties/url" for each manifest

**Review by Justkant — APPROVED** [review]

**Pull request merged**